app/models/copy_staging/serialize_work.rb

```diff
@@ -85,5 +82,14 @@ def shrine_config(shrine_storage_key)
         "prefix" => storage.prefix
       }
     end
+
+    # The list of Shrine storages we can copy from and to
+    # is listed as a constant in
+    # CopyStaging::RestoreWok, so let's use that here.
+    def storages
+      storage_list = CopyStaging::RestoreWork::ORIGINALS_STORAGE + CopyStaging::RestoreWork::DERIVATIVES_STORAGE
+      Hash[ storage_list.collect { |v| [v.to_s, shrine_config(v)  ] } ]
```

Comment:
Again all this indirection is seeming. bit confusing ot me, I honestly would have just written it out here, even if it duplicates things that are other places. Becuase then the develoepr could see exactly what's going on, right now... if I come back to this and want to know what's going on, it's going to take me a while to go back and forth between the multiple data structures in multiple files and figure out what the heck is going on.

Reply:
Good call -- -added some more documentation to clear this up a bit.